Cloudinary/Classes/Core/Network/CLDNetworkCoordinator.swift

```diff
@@ -26,11 +26,11 @@ import Foundation
 import UIKit
 
 internal class CLDNetworkCoordinator: NSObject {
+
+    static let DEFAULT_VERSION =        "3.3.0"
```

Comment:
Why not expose `func getVersion()` instead?

Reply:
Checking

Reply:
Internal class, have to expose it somehow to Analytics class to get the current version of the SDK